src/seedu/addressbook/ui/TextUi.java

```diff
@@ -88,24 +90,16 @@ public String getUserCommand() {
         return fullInputLine;
     }
 
-
     public void showWelcomeMessage(String version, String storageFilePath) {
         String storageFileInfo = String.format(MESSAGE_USING_STORAGE_FILE, storageFilePath);
-        showToUser(
-                DIVIDER,
-                DIVIDER,
-                MESSAGE_WELCOME,
-                version,
-                MESSAGE_PROGRAM_LAUNCH_ARGS_USAGE,
-                storageFileInfo,
+        showToUser(DIVIDER, DIVIDER, MESSAGE_WELCOME, version, MESSAGE_PROGRAM_LAUNCH_ARGS_USAGE, storageFileInfo,
                 DIVIDER);
     }
```

Comment:
Oops, missed out this one. The `DIVIDER` of line 96 should match the first `DIVIDER` of line 95, am I right?